OTAnalytics/plugin_ui/customtkinter_gui/line_section.py

```diff
@@ -112,31 +180,25 @@ def __init__(
         self._is_finished = is_finished
```

Comment:
Add the style for temporary lines (selected & edited) as a instance variable/constructor parameter

Reply:
Add the style for temporary lines as a instance variable/constructor parameter here and also in `SectionBuilder`.